vendor.conf

```diff
@@ -40,7 +40,7 @@ github.com/gofrs/flock                              392e7fae8f1b0bdbd67dad7237d2
 # libnetwork
 
 # When updating, also update LIBNETWORK_COMMIT in hack/dockerfile/install/proxy.installer accordingly
```

Comment:
Is this comment still accurate / doesn't this PR need to include an update to `LIBNETWORK_COMMIT` in `hack/dockerfile/install/proxy.installer`? 😅

Reply:
Why am I seeing this when it's FINALLY green? :D